sdks/java/testing/nexmark/src/main/java/org/apache/beam/sdk/nexmark/queries/BoundedSideInputJoinModel.java

```diff
@@ -76,6 +76,6 @@ public BoundedSideInputJoinModel(NexmarkConfiguration configuration) {
 
   @Override
   protected Collection<String> toCollection(Iterator<TimestampedValue<Bid>> itr) {
-    return toValueTimestamp(itr);
+    return toValue(itr);
```

Comment:
@apilloud I think you did the most on these models. In SQL the timestamp itself is not really observable. The next GROUP BY will have some windowing expression (or not) and the timestamp is specified according to the arguments to the windowing function.